test/internals/initialConfig.test.js

```diff
@@ -0,0 +1,245 @@
+'use strict'
+
+const { test } = require('tap')
+const Fastify = require('../..')
+const fs = require('fs')
+const path = require('path')
+const http = require('http')
+const pino = require('pino')
+const split = require('split2')
+const deepClone = require('rfdc')({ circles: true, proto: false })
+
+test('Fastify.initialConfig is an object', t => {
+  t.plan(1)
+  t.type(Fastify().initialConfig, 'object')
+})
+
+test('without options passed to Fastify initialConfig is an empty object', t => {
```

Comment:
I think that this object should always expose something, in this case, for example, it should expose the default options.
@mcollina what do you think?

Reply:
+1

Reply:
done ^^